docs/server-config.md

```diff
@@ -4,47 +4,149 @@
 
 <!-- toc -->
 
+- [Custom Settings](#custom-settings)
+  * [Configuration File Content](#configuration-file-content)
+  * [Configuration File Example](#configuration-file-example)
 - [Configuring HTTP vs HTTPS](#configuring-http-vs-https)
 - [Authentication](#authentication)
   * [Enabling Authentication](#enabling-authentication)
   * [Password File](#password-file)
-  * [Setting Up Environment](#setting-up-environment)
   * [Adding an Account](#adding-an-account)
   * [Updating an Account](#updating-an-account)
   * [Deleting an Account](#deleting-an-account)
 - [Logging](#logging)
   * [Request Context Logging](#request-context-logging)
-- [Custom Script Timeout](#custom-script-timeout)
 
 <!-- tocstop -->
 
 <!-- markdownlint-enable MD004 -->
 
-Default settings for TabPy may be viewed in the
-tabpy_server/common/default.conf file. This file also contains
-commented examples of how to set up your TabPy server to only
-serve HTTPS traffic and enable authentication.
+## Custom Settings
 
-Change settings by:
+TabPy starts with set of default settings unless settings are provided via
+environment variables or with a config files.
 
-1. Adding environment variables:
-   - set the environment variable as required by your Operating System. When
-     creating environment variables, use the same name as is in the config file
-     as an environment variable. The files startup.sh and startup.cmd in the root
-     of the install folder have examples of how to set environment variables in
-     both Linux and Windows respectively. Set any desired environment variables
-     and then start the application.
-2. Modifying default.conf.
-3. Specifying your own config file as a command line parameter.
-   - i.e. Running the command:
+Configuration parameters can be updated with:
 
-     ```sh
-     python tabpy.py --config=path\to\my\config
-     ```
+1. Adding environment variables - set the environment variable as required by
+   your Operating System. When creating environment variables, use the same
+   name as is in the config file as an environment variable.
+2. Specifying a parameter in a config file (enviroment variable value overwrites
+   configuration setting).
+
+Configuration file with custom settings is specified as a command line parameter:
+
+```sh
+tabpy --config=path/to/my/config/file.conf
+```
 
 The default config file is provided to show you the default values but does not
 need to be present to run TabPy.
 
+### Configuration File Content
+
+Configuration file consists of settings for TabPy itself and Python logger
+settings. You should only set parameters if you need different values than
+the defaults.
+
+TabPy parameters explained below and for the logger documentation
```

Comment:
TabPy parameters explained below, the logger documentation can be found at

Reply:
fixed